File: 0724-find-pivot-index/ganguklee.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        l = 0
        r = sum(nums)

        for i, x in enumerate(nums):
            if i-1 >= 0:
                l += nums[i-1]
            r -= x
            if l == r:
                return i
            
        return -1

File: 0001-two-sum/ganguklee.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        for (i, n) in nums.enumerated() {
            for (j, m) in nums.enumerated() {
                if i == j {
                    continue
                }

                if n+m == target {
                    return [i, j]
                }
            }
        }

        return []
    }
}